packages/frontend/app/styles/components/messages.css

```diff
@@ -32,6 +32,11 @@
   }
 }
 
+a.contact-display{
```

Comment:
since the work here is in the sidebar, should the css for the sidebar go with the sidebar styles? 
this stylesheet is intended for the message list in the chat history view